modules/building/pages/creating-secrets.adoc

```diff
@@ -60,7 +60,9 @@ Some container builds may use parent images from registries that require authent
 . Click on `Secrets` on the left menu.
 . Click on `Add Secret`
 . Choose `Image pull secret` for `Secret type`
-. Enter `Secret name` , for example `my-quay-secret` 
+. Enter `Secret name` , for example `my-quay-secret`
+. From `Link secret options` select either `All existing and future components in the namespace` or
+  `Select components in the namespace` and specify a component
```

Comment:
It's possible to link to several components from the list

Reply:
ok